### PR TITLE
gtk-play: Modify switch statement in stream_info_get_string

### PR DESCRIPTION
**Pull request by JimmyOhn**
In stream_info_get_string function, buffer variable is used all switch case repeatedly. Also, return used in switch statement. So, I have modified that cases.

### CODE REVIEW

File: gst-play/gst-play.c [hunk 1]
```diff
@@ -195,9 +195,12 @@ static void
 print_all_stream_info (GstPlayerMediaInfo * media_info)
 {
   guint count = 0;
-  GList *l, *list;
+  GList *list, *l;
 
   list = gst_player_media_info_get_stream_list (media_info);
+  if (!list)
+    return;
```
Comment (sdroege) [inline gst-play/gst-play.c]: Even if we have no stream list here, we could at least print the URI, duration and other stream-list-independent things below.

**Comment by sdroege** [comment]
Can you update your pull request?

**Comment by JimmyOhn** [comment]
Sorry for my late reply. I got back from vacation. I'll update ASAP.

**Comment by sdroege** [comment]
Sure, no worries :)

**Comment by JimmyOhn** [comment]
I have modified above code for your comment.:)

**Comment by sdroege** [comment]
Looks good now but can you squash together the commits? One for the list stuff in gst-play, another for the gtk-play parts?

**Comment by JimmyOhn** [comment]
I have modified commits:)

**Comment by sdroege** [comment]
Thanks, merged

**Pull request closed without merging**